packages/chart-advisor/src/ruler/interface.ts

```diff
@@ -160,3 +108,22 @@ export type DesignRuleModule = DefaultRuleModule & {
   type: 'DESIGN';
   optimizer: Optimizer;
 };
+
+export type RuleModule = ChartRuleModule | DesignRuleModule;
+
+/**
+ * @public
+ * rule config
+ */
+export type RuleConfig = {
+  include?: string[];
```

Comment:
Should  type of `include` and `exclude` be limited to `ChartRuleID[]` ? Or we can use `string[]` here and add a note to explain is an array of chart rule ids ?

Reply:
rule id could be customized, too..will add a comment

Reply:
done